src/WpfMath/TexPredefinedFormulaParser.cs

```diff
@@ -1,341 +1,339 @@
-using System;

```

Comment:
Previously this file had `CRLF` and now it has `LF` (it was replaced because I renamed it to match the class name). I think that's alright.

Please check the online diff with `?w=1` appended to the page URL to see the real changes (it will ignore the whitespace diff): https://github.com/ForNeVeR/wpf-math/pull/123/files?w=1#diff-e81f0c498a5c8d83d49d999c0375f718